src/pages/NewChatPage.tsx

```diff
@@ -262,7 +264,7 @@ function NewChatPage({betas, isGroupChat, personalDetails, reports, isSearchingF
                             headerMessage={headerMessage}
                             boldStyle
                             shouldPreventDefaultFocusOnSelectRow={!DeviceCapabilities.canUseTouchScreen()}
-                            shouldShowOptions={isOptionsDataReady && didScreenTransitionEnd}
+                            shouldShowOptions={areOptionsInitialized && didScreenTransitionEnd}
```

Comment:
```suggestion
                            shouldShowOptions={areOptionsInitialized}
```
This will be enough right, because we have this above:
```javascript
    const {options, areOptionsInitialized} = useOptionsList({
        shouldInitialize: didScreenTransitionEnd,
    });
```

and inside our `useOptionsList`, we won't `initializeOptions()` if below conditions passed:
```
        if (!shouldInitialize || areOptionsInitialized) {
            return;
        }
```